src/NuGet.Core/NuGet.Indexing/project.json

```diff
@@ -0,0 +1,45 @@
+﻿{
+  "version": "3.4.0-*",
+  "description": "NuGet.Indexing Class Library",
+  "compile": [
+    "../../../submodules/NuGet.Services.Metadata/src/NuGet.Indexing/PackageAnalyzer.cs",
+    "../../../submodules/NuGet.Services.Metadata/src/NuGet.Indexing/NuGetQuery.cs",
+    "../../../submodules/NuGet.Services.Metadata/src/NuGet.Indexing/IdentifierKeywordAnalyzer.cs",
+    "../../../submodules/NuGet.Services.Metadata/src/NuGet.Indexing/IdentifierAnalyzer.cs",
+    "../../../submodules/NuGet.Services.Metadata/src/NuGet.Indexing/ShingledIdentifierAnalyzer.cs",
+    "../../../submodules/NuGet.Services.Metadata/src/NuGet.Indexing/DescriptionAnalyzer.cs",
+    "../../../submodules/NuGet.Services.Metadata/src/NuGet.Indexing/DotTokenizer.cs",
+    "../../../submodules/NuGet.Services.Metadata/src/NuGet.Indexing/TokenizingHelper.cs",
+    "../../../submodules/NuGet.Services.Metadata/src/NuGet.Indexing/IdentifierAutocompleteAnalyzer.cs",
+    "../../../submodules/NuGet.Services.Metadata/src/NuGet.Indexing/OwnerAnalyzer.cs",
+    "../../../submodules/NuGet.Services.Metadata/src/NuGet.Indexing/VersionAnalyzer.cs",
+    "../../../submodules/NuGet.Services.Metadata/src/NuGet.Indexing/CamelCaseFilter.cs",
+    "../../../submodules/NuGet.Services.Metadata/src/NuGet.Indexing/TagsAnalyzer.cs",
+    "../../../submodules/NuGet.Services.Metadata/src/NuGet.Indexing/SemanticVersionFilter.cs"
+  ],
+  "dependencies": {
+    "EntityFramework": "5.0.0",
+    "Lucene.Net": "3.0.3",
+    "Lucene.Net.Contrib": "3.0.3",
```

Comment:
We can't pull in lucene.net.contrib or lucene.net.store.azure, also not sharpziplib or storage into the client for tis